apps/web/src/components/modals/failed-ping-alert-confirmation.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+} from "@openstatus/ui";
+
+import { LoadingAnimation } from "@/components/loading-animation";
+import type { MonitorProps } from "../forms/montitor-form";
+
+type FailedPingAlertConfirmationProps = {
+  submit: ({ ...props }: MonitorProps) => void;
+  props: MonitorProps;
+  pingFailed: boolean;
+  setPingFailed: React.Dispatch<React.SetStateAction<boolean>>;
+  loading: boolean;
```

Comment:
We don't have to pass `loading` here. And I am not sure if it gets correctly updated as the `loading` state is binded on the parent `onSubmit` function which won't be triggered when clicking on "Confirm" here.

Instead, create your own `[isPending, startTransition]` in this file and wrap the function inside it. Similar to the `monitor-form.tsx`.